lightning_pose/models/base.py

```diff
@@ -367,13 +367,13 @@ def evaluate_labeled(
         # compute and log loss on labeled data
         loss, log_list = self.loss_factory(stage=stage, **data_dict)
 
-        # compute and log rmse loss on labeled data
+        # compute and log pixel_error loss on labeled data
```

Comment:
what makes the actual loss different now that we call it "pixel error"? it's the same rmse computation no?